pkg/storage/module/upload.go

```diff
@@ -43,6 +44,8 @@ func Upload(ctx context.Context, module, version string, info, mod, zip io.Reade
 			errChan <- fmt.Errorf("uploading %s.%s.%s failed: %s", module, version, ext, tctx.Err())
 		}
 	}
+
+	runtime.GC()
```

Comment:
why is this necessary?

Reply:
Not required at all. This was to test what happens on Windows when I GC